index.html

```diff
@@ -39,108 +45,156 @@ <h1><span class="first-part">I'm Tonny Tei,</span><br><span class="second_part">
          </div>
 
          <!-- Page five -->
-         <div class="left-block">
-            <div class="primary-image">
-                <img src="images/project4.png" alt="photo_of_my_projects">
-            </div>
-            <div class="title">
-                <h1>Multi-post <br> Stories</h1>
+         <div class="left-block" id="page-five">
+            <div class="primary-image" id="grid-seven">
+                <img src="images/project4.png" alt="photo_of_my_projects" class="for-small-size">
+                <img src="images/Snapshoot Portfolio-four.png" alt="Snapshoot_portfolio" class="for-big-size">
             </div>
-            <div class="frame-2">
-                <ul>
-                    <li>CANOPY</li>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                </ul>
-            </div>
-            <p class="primary-text">Daily selection of privately personalised reads" no accounts or signups required. </p>
-            <div class="tags"> 
-                <ul>
-                    <li>html</li>
-                    <li>CSS</li>
-                    <li>javascript</li>
-                </ul>
-            </div>
-            <div class="action"> <a href="#"> See project</a> </div>
+
+            <div id="grid-eight" class="right">
+                <div class="title">
+                    <h1 class="for-small-size">Multi-post <br> Stories</h1>
+                    <h1 class="for-big-size"> Uber Navigation </h1>
+                </div>
+                <div class="frame-2">
+                    <ul class="for-small-size">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+
+                    <ul class="for-big-size">
+                        <li>Uber</li>
+                        <li>Lead Developer</li>
+                        <li>2018</li>
+                    </ul>
+                </div>
+                <p class="for-small-size">Daily selection of privately personalised reads" no accounts or signups required. </p>
+                <p class="for-big-size">A smart assistant to make driving more safe, efficient, and fun by unlocking 
+                    your most expensive computer: your car.</p>
+                <div class="tags"> 
+                    <ul class="for-small-size">
+                        <li>html</li>
+                        <li>CSS</li>
+                        <li>javascript</li>
+                    </ul>
+                    <ul class="for-big-size">
+                        <li>html</li>
+                        <li>Rubby on Rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                    </ul>
+                </div>
+                <div class="action"> <a href="#"> See project</a> </div>
          </div>
+        </div>
     </section>
     <!-- About page: sixth page -->
     <section class="about-block">
```

Comment:
- [ ] Kindly ensure that the color of all your about section text matches your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/194626052-29c37e9c-508f-42c0-a6cb-b10b33c1c183.png)  | ![image](https://user-images.githubusercontent.com/63473584/194626172-5b639e71-c04f-4d86-9afb-55d815b9b527.png)  |